deploy_hpo.sh

```diff
@@ -28,16 +28,19 @@ HPO_CONTAINER_IMAGE=${HPO_REPO}:${HPO_VERSION}
 setup=1
 cluster_type="native"
 CONTAINER_RUNTIME="docker"
+service_type="both"
 
 # source the helpers script
 . ${SCRIPTS_DIR}/cluster-helpers.sh
 
 function usage() {
 	echo
 	echo "Usage: $0 [-c [docker|minikube|native]] [-o hpo container image]"
-	echo "       -s = start(default), -t = terminate"
+    echo " -s = start(default), -t = terminate"
 	echo " -c: cluster type."
```

Comment:
Almost all the changes have tab / space issues. As mentioned previously, we use tabs for all scripts. I think there is a IDE option to set this locally, easier that way